system76-firmware/fix-riscv64-io.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/system76-firmware/daemon/src/main.rs b/system76-firmware/daemon/src/main.rs
+index 59ee5bf..065f572 100644
+--- a/system76-firmware/daemon/src/main.rs
++++ b/system76-firmware/daemon/src/main.rs
+@@ -1,10 +1,13 @@
+ use dbus::blocking::Connection;
+ use dbus_crossroads::{Context, Crossroads, MethodErr};
+-use std::{io, process};
++use std::{process};
+ 
+ use system76_firmware::*;
+ use system76_firmware_daemon::*;
+ 
++#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
++use std::{io};
```

Comment:
Please leave some notes here. Why is it only used on x86? Should the change be upstreamed also?